Automattic/wp-calypso
# Sites Management Page: fix fetch error

Fixes https://github.com/Automattic/wp-calypso/issues/79032

## Proposed Changes

This pr adds a fix for `Failed to fetch` error that occurs in `checkRedirectImage` function.

## Testing Instructions
1. Apply the following diff:
```
diff --git a/packages/components/src/site-thumbnail/index.tsx b/packages/components/src/site-thumbnail/index.tsx
index 590fb0c046..fa7fd9a898 100644
--- a/packages/components/src/site-thumbnail/index.tsx
+++ b/packages/components/src/site-thumbnail/index.tsx
@@ -49,7 +49,7 @@ export const SiteThumbnail = ( {
 		h: height,
 		...mshotsOption,
 	};
-	const { imgProps, isLoading, isError } = useMshotsImg( mShotsUrl, options, [
+	const { imgProps, isLoading, isError } = useMshotsImg( 'example.com', options, [
 		...dimensionsSrcset,
 		{ width, height },
 	] );
```
2. Open dev tools
3. Visit `/sites`
4. Follow the instructions below:

![sites_error_network](https://github.com/Automattic/wp-calypso/assets/497103/95581f6a-364d-41e5-bdd9-7254d716b2ec)

5. Ensure that the error is no longer shown



*

## Pre-merge Checklist
